std/typecons.d

```diff
@@ -8296,9 +8281,7 @@ if (is(T == class))
 
         ~this()
         {
-            // `destroy` will also write .init but we have no functions in druntime
-            // for deterministic finalization and memory releasing for now.
-            .destroy(Scoped_payload);
+            .destroy!false(Scoped_payload);
```

Comment:
Why you changed this tho?

Reply:
See the comment - we *can* skip the useless re-initialization. Looking into this a bit further led to https://github.com/dlang/druntime/pull/3821... ;)